jdaviz/configs/imviz/helper.py

```diff
@@ -285,6 +478,8 @@ def load_static_regions(self, regions, **kwargs):
             #                  'Consider using a different region name.')
             #    continue
```

Comment:
Is now a good time to incorporate the two `TODO` chunks above since it seems this PR should fix the issue to which they refer?

Reply:
No, I think it is too late because this method is now deprecated. No one is supposed to use it anymore. It is not removed outright for backward compatibility but eventually we will remove this altogether.